rpcs3/rpcs3qt/progress_dialog.cpp

```diff
@@ -2,10 +2,12 @@
 
 #include <QLabel>
 
-progress_dialog::progress_dialog(const QString &labelText, const QString &cancelButtonText, int minimum, int maximum, QWidget *parent, Qt::WindowFlags flags)
+progress_dialog::progress_dialog(const QString& windowTitle, const QString &labelText, const QString &cancelButtonText, int minimum, int maximum, QWidget *parent, Qt::WindowFlags flags)
 	: QProgressDialog(labelText, cancelButtonText, minimum, maximum, parent, flags)
 {
+	setWindowTitle(windowTitle);
```

Comment:
I don't really get why this was necessary (not against it but it seems very random)

Reply:
I mean it just makes sense to have the title of the dialog set in the constructor.